### DISCUSSION TITLE
What does blue "Time ..." output mean in terminal

### DISCUSSION THREAD

**Discussion by abstractpoint** [vercel/next.js#41138] [Help] [answered]
This seems to be either undocumented or very hard to find the answer to, and it only appears when Next.js project grows beyond a basic app. 

What does blue "Time ..." output mean in terminal? Is it a time to first byte when page loads? What determines this appearance, and how to turn that off?

<img width="570" alt="Screenshot 2022-10-03 at 19 59 32" src="https://user-images.githubusercontent.com/3032975/193659904-5a82b8be-7276-4424-a87b-ad129cef8773.png">

Many thanks!

**Comment by icyJoseph** [comment]
What terminal emulator are you using btw? 

This does look very strange though. Specially because most messages in the code base use a wrapper around chalk, which does `coloured-label` + `un-coloured-content` kind of pattern, for example `info - automatically...` so to see `Time <float.number>` without units is rare. 

Could it be a package you are using that's measuring time differences and logging?

**Comment by gaston-flores** [comment, marked as answer]
This appears because you enabled `optimizeCss`: This is the output of [critters](https://github.com/GoogleChromeLabs/critters/blob/ec6e1443a75e915e81f4e4ee423cb149c2ae1f50/packages/critters/src/index.js#L194), not Next.js. You can conditionally disable `optimizeCss` for development if you don't want to see this.

**Reply by abstractpoint** [reply]
This was it, thank you so much.
